pkg/controller/apiserver/apiserver_controller.go

```diff
@@ -175,51 +182,66 @@ func (r *ReconcileAPIServer) Reconcile(ctx context.Context, request reconcile.Re
 			r.status.OnCRNotFound()
 			return reconcile.Result{}, nil
 		}
-		r.status.SetDegraded(msg, err.Error())
-		reqLogger.Error(err, fmt.Sprintf("An error occurred when querying the APIServer resource: %s", msg))
+		r.SetDegraded(operatorv1.ResourceReadError, fmt.Sprintf("An error occurred when querying the APIServer resource: %s", msg), err, reqLogger)
 		return reconcile.Result{}, err
 	}
 	r.status.OnCRFound()
 	reqLogger.V(2).Info("Loaded config", "config", instance)
 
+	//Set the meta info in the tigerastatus like observedGenerations
```

Comment:
nit: It's best to make sure all comments are consistent throughout the PR:
 - "//" followed by a space
 - Start with Capital and end with "."
 - Capitalize resource objects like ApiServer and Tigerastatus.

Reply:
Done